tests/e2e/linalg/BUILD.bazel

```diff
@@ -35,6 +35,10 @@ iree_check_single_backend_test_suite(
     name = "check_llvm-cpu_local-task",
     srcs = LLVM_SRCS,
     driver = "local-task",
+    tags = [
+        # subbyte support for wasm is not on priorities.
+        "nowasm",
+    ],
```

Comment:
@ScottTodd I want to disable the test suite for wasm, but I failed. How do I disable it properly?

https://github.com/openxla/iree/actions/runs/8745270346/job/24001316668?pr=17074

Reply:
Hmmmm... I think this is only disabling the runtime portion (which doesn't exist on wasm anyways): https://github.com/openxla/iree/blob/f755b422892bfb8ebb2ee8258f9aac140cddbeb4/build_tools/cmake/iree_check_test.cmake#L135-L143

CMake configuration whack-a-mole... we could change that logic again, or we could set `-DIREE_BUILD_ALL_CHECK_TEST_MODULES=OFF` in the cross compile scripts. Let me try that and confirm that the expected sets of tests are still compiling/running. Looking at the logs, the emscripten build is compiling CUDA tests, when it really doesn't need to :P

Reply:
Ah, it's end of day actually, and tomorrow might be busy with the organization move. How urgent is this?

I'd basically want to set that option in https://github.com/openxla/iree/blob/main/build_tools/cmake/build_runtime_emscripten.sh (and https://github.com/openxla/iree/blob/main/build_tools/cmake/build_riscv.sh) like how it is already set in https://github.com/openxla/iree/blob/main/build_tools/cmake/build_android.sh, then look at the CI logs to see that all check tests for those targets are still being built.

Then for this PR, setting nowasm should I think exclude this particular list of tests again.

Reply:
It is not urgent. I can wait. Thanks for taking a look at it!

Reply:
I think https://github.com/iree-org/iree/pull/17114 will help (without sacrificing too much coverage...)